odbc_fdw.c

```diff
@@ -1343,6 +1336,8 @@ odbcBeginForeignScan(ForeignScanState *node, int eflags)
 static TupleTableSlot *
 odbcIterateForeignScan(ForeignScanState *node)
 {
+    EState	   *estate = node->ss.ps.state;
```

Comment:
is that a suggestion for the variable name? or just  asking about the meaning of the state data?